README.md

```diff
@@ -13,13 +13,12 @@ To execute scripts that use `mex-wholebodymodel`, make sure that this directorie
 An example on how to use mex-wholebodymodel to compute the dynamics quantities of 
 a rigid body is available at [examples/rigidBodyDynamics.m](examples/rigidBodyDynamics.m).
 
-
-## Controller simulation
-In the [matlab-src/mex-wholebodymodel_balancing](matlab-src/mex-wholebodymodel_balancing/) and
-[matlab-src/TorqueBalancing_js](matlab-src/TorqueBalancing_js/) directories you can find two different
-simulations of a whole-body controller implemented using the mex-wholebodymodel interface.
-For more information please check the relative [README](matlab-src/mex-wholebodymodel_balancing/README),
-[README](matlab-src/TorqueBalancing_js/README).
+## Controllers simulations
+In [matlab-src/torqueBalancing_matlab](matlab-src/torqueBalancing_matlab/) the user can find two different
```

Comment:
Here we use "the user" why later we use "you". Let's be consider and always use you.